time_stamper_cv/include/image_processor.h

```diff
@@ -27,6 +29,12 @@ class ImageProcessor {
    */
   void setVisualization(bool visualization);
 
+  /**
+   * Message to be published
+   * @return Led state with set values
+   */
+  time_stamper_cv::Ledstate getLedStateMessage();
```

Comment:
I'd reserve `get` for getters. This function isn't a getter at all because it doesn't just _get_ something that is already there. What about `build` ?